pkg/network/controllers/hotplug_test.go

```diff
@@ -128,12 +114,6 @@ var _ = Describe("Network interface hot{un}plug", func() {
 			[]v1.Interface{{Name: "blue"}, {Name: "red"}},
 			[]v1.Network{{Name: "blue"}, {Name: "red"}},
 		),
-		Entry("should remain, given absent iface and associated status ifaces (i.e.: unplug pending)",
```

Comment:
Where is this covered now? (absent remains absent)

Reply:
Good catch! I was under the impression that is was covered by another test - but it wasn't.
Added the missing coverage.